sdk/golang-example/hello world.txt

```diff
@@ -0,0 +1,50 @@
+### (Environment setup)

```

Comment:
What's the purpose of this file? It seems to be an abbreviated version of `fibonacci.txt`.

Reply:
this is actually example of hello world program and another one is fibonacci , difference is in the code area and rest all instructions are same

Reply:
> this is actually example of hello world program and another one is fibonacci , difference is in the code area and rest all instructions are same

Probably better to put the instructions in `sdk/go-examples/README.md`. We already have `sdk/c-examples/README.md`, so there's a precedent.

And perhaps we don't need the Fibonacci example if it doesn't use any Veracruz facility that the "Hello, world" examples doesn't use?

Reply:
actually the issue was like "More languages can be compiled to the WASM binaries now. It is worth exploring and providing a guide to compile more languages to WASM/WASI binaries" so i talked to my mentor Shale Xiong regarding these examples fibonacci and hello-world and he said dont have any issue with that , so i worked upon it

Please let me know if i am mistaking somewhere